rotkehlchen/chain/ethereum/modules/yearn/vaults.py

```diff
@@ -184,6 +184,8 @@ def get_usd_price_zero_if_error(
     inquirer = Inquirer()
     if asset in inquirer.special_tokens:
         return inquirer.find_usd_price(asset)
+    if isinstance(asset, EthereumToken) and asset.protocol == 'yearn_v2_vault':
+        return inquirer.find_usd_price(asset)
```

Comment:
Why another if branch? The body is the same as the above, so should be put in there